### PR TITLE
replace use of `cl_oo_abap_doc_reader`

### PR DESCRIPTION
**Pull request by larshp**
#160
#108

### CODE REVIEW

**Comment by larshp** [comment]
its alive! 👽 

image

File: abap_transpile.json [hunk 1]
```diff
@@ -44,6 +44,8 @@
       {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "get_abap_doc_for_absolute_name", "note": "Void type: cl_oo_abap_doc_reader"},
       {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "stack_stores_operations", "note": "??"},
       {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "get_type_info_string_like_enum", "note": "enums are not yet supported"},
+      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "get_type_info_string_like_enum", "note": "enums are not yet supported"},
```
Comment (schneidermic0) [inline abap_transpile.json:47]: This line is duplicated, isn't it?

File: src/zcl_aff_writer.clas.abap [hunk 2]
```diff
@@ -430,11 +430,11 @@ CLASS zcl_aff_writer IMPLEMENTATION.
 
 
   METHOD call_reader_and_decode.
-    DATA(reader) = NEW cl_oo_abap_doc_reader( ).
+    DATA(ref) = cl_oo_factory=>create_instance( )->create_clif_source( name_of_source ).
+    ref->get_source( IMPORTING source = DATA(lt_source) ).
+    DATA(reader) = zcl_aff_abap_doc_reader=>create_instance( lt_source ).
```
Comment (schneidermic0) [inline src/zcl_aff_writer.clas.abap:435]: Can we remove the prefix of the local variable? We usually don't use any prefixes in this repo.

```suggestion
    ref->get_source( IMPORTING source = DATA(source) ).
    DATA(reader) = zcl_aff_abap_doc_reader=>create_instance( source ).
```

**Review by schneidermic0 — CHANGES_REQUESTED** [review]
Cool 👍 
Thank you!

I have just two small comments...

File: abap_transpile.json (hunk 1 above)
Comment (larshp) [reply abap_transpile.json]: thanks, fixed

**Review by schneidermic0 — APPROVED** [review]
LGTM. Thanks, again.

**Pull request merged**